includes/class-wp-job-manager-ajax.php

```diff
@@ -214,6 +149,27 @@ public function get_listings() {
 
 		$result['max_num_pages'] = $jobs->max_num_pages;
 
+		// Give plugins the opportunity to control what JSON is returned
+		if ( $result = apply_filters( 'wp_job_manager_ajax_get_jobs', null, $result, $jobs ) ) {
```

Comment:
The majority of filters use `job_manager_*` as their name, we should keep this consistent.